### DISCUSSION TITLE
Dynamite Core Team Meeting - Nov 8th

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#416] [Announcements]
Dear @dynamics-of-stellar-systems/dynamite-maintainers,

we will resume our every other week DYNAMITE core team meetings on **Nov 8th at 1pm in Glenn's office or the zoom link** (see moodle/discord). I also created a webhook, so now once I send this, it hopefully get posted in the dynamite channel on our new discord server. I think this is nice to decrease email spam to the maintainers and also make it more transparent to non-core members what we are up to and they have the opportunity to join the meeting if they find it interesting.

I hope this is ok, any objections/suggestions are welcome!

Cheers,
Alina

**Comment by boeckera** [comment]
@dynamics-of-stellar-systems/dynamite-maintainers:
**Reminder, this Friday! (React if you are planning to join please :))**

@maindlt : would you be willing to update us on what has been going on with the recent issues ?
We could also discuss the binning of the orbits for the coloring!

Please also welcome @FranciscoAros to the team :)

**Reply by maindlt** [reply]
Yes sure, I'd love to get several of the PRs out of the way, I think we can merge most of them right in the meeting.
Coloring: there is also a PR on the binning. As it is from two months ago, I need to refresh my memory of the details and open questions (hopefully I'll succeed 😂)...
Welcome @FranciscoAros! 🙂

**Comment by afeldmei** [comment]
@dynamics-of-stellar-systems/dynamite-maintainers 
Two weeks ago, we agreed to meet again today (November 22)! Sorry for the late reminder! I hope to see many of you later. Please react if you will join

**Comment by prashjet** [comment]
We decided to meet next week 13:15 next Friday to run through the checklist for the code release.

**Reply by maindlt** [reply]
Hi @prashjet,
while testing for the release, I ran into a strange error: while test_nnls.py executes as expected, tutorial 2 (calculate a single model) freezes when using the Python NNLS and crashes when using LegacyWeightSolver. Wild guess: it might be connected to @afeldmei's troubles with the weight solving.
Shall we postpone our meeting until this issue is solved?

**Reply by prashjet** [reply]
Hi Thomas. Good spot! Yes I guess we should postpone in that case. It would be excellent if this also solved the problems Anja has been running into. 

Please let us know if you need any input to help solve this.

FYI I will be away next week Tuesday-Friday, back the following week.

**Reply by maindlt** [reply]
Hi Prash, many thanks, I am fine for now (still figuring out what exactly happens and why) :-)
Have a great SFB meting next week - shall we meet for the release on Fri, Dec 13, 13:15 then? The rest of us can use next Fri for an update on proper motions...
Cheers, Thomas
